src/core/lightning/wallet.ts

```diff
@@ -35,7 +35,7 @@ export class LightningUserWallet extends OnChainMixin(UserWallet) {
       logger: this.logger,
     })
 
-    return redlock({ path: this.user._id, logger: this.logger }, async () => {
+    return redlock({ path: this.user.id, logger: this.logger }, async () => {
```

Comment:
Will this change be an issue if there is a lock inflight during upgrade?

Reply:
`_id` and `.id` are the same value, so no.

Reply:
afaik one of them is a string serizliazation and the other is an object